### DISCUSSION TITLE
Any possibility to create uninstaller which works without system on ?

### DISCUSSION THREAD

**Discussion by RandomUsername12222** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#111] [Q&A] [answered]
Hi there Chris. 
I would like to ask if there is any way to "manually" remove your wifi drivers from system? I was using your drivers in version 7 but after time my dongle got unresponsive so I wanted to update. After update to v10 ( without uninstall ), system is stuck on apple logo :/ . I can connect to macos drive from linux. Any help would be appreciated since I have around 160gb of data and nowhere to copy it :(

I am using pc if that matters.

Huge thank you if you can help <3

**Comment by chris1111** [comment, marked as answer]
The WIFI drivers is not on the system You are using the OC Version so the Drivers is in the EFI partition in /EFI/OC/kext

**Reply by RandomUsername12222** [reply]
Thx for answer. I found that kext, removed it and device seems to be booting now. Sorry for taking your time :P
